pages/1.11/release-notes/1.11.5/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.5
+title: Release Notes for 1.11.5
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.5
+---
+
+DC/OS 1.11.5 was released on September 12, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# Notable Changes in DC/OS 1.11.5
```

Comment:
This is better, @anithamuthu79  - This is consistent with what we had for 1.11.4

Reply:
Sure.